src/page-response-impact/utils/ResponseImpactCardStateUtils.tsx

```diff
@@ -3,9 +3,24 @@ import styled from "styled-components";
 import Colors from "../../design-system/Colors";
 import {
   countEverHospitalizedForDay,
+  countUnableToWorkForDay,
   getFatalitiesForDay,
+  getHospitalizedForDay,
 } from "../../impact-dashboard/ImpactProjectionTableContainer";
 import { CurveData, isCurveData } from "../../infection-model";
+import { calculatePercentDiff } from "./numberUtils";
+
+export function maxByIndex(twoDimensionalArray: number[][]) {
+  let sumByIndex = [];
+  for (let i = 0; i < 90; i++) {
```

Comment:
I wonder about hard coding this to 90. We actually ahve this number in `sumPopulationImpactAcrossFacilities` as `incarceratedData.shape[0]` and `staffData.shape[0]` (`-1`?) and could pass it into this new function. Might be clean to assign its own variables for `staffLastDay` and `incarceratedLastDay` or something

Reply:
Totally agree, good suggestion. I'll do that now.

Reply:
@jovergaag I actually have this as a constant in `responseChartData`